Project-3/Jvonderh Project 3

```diff
@@ -0,0 +1,228 @@
+  (d)  Functional 
+  Characteristic of
+  This should not be functional because, in a relationship between a characteristic and its bearer where the characteristic depends on the bearer for its existence, it is possible for a characteristic to have multiple bearers. Since a functional object property requires that it only output only one possible bearer in this case, it would contradict the property for it to have multiple bearers. As an example, the characteristic "weighs 50 pounds" will have multiple bearers. This may include boxes of office supplies, a piece of furniture, etc.,  but it will mean that this characteristic is a characteristic of multiple bearers. This means that there is more than one value for the characteristic of relation, meaning that it should not be functional.
+  (e)  Inverse Functional
+  Has characteristic
+  This should not be inverse functional because, in a relationship where a bearer has a characteristic which depends on the bearer for its existence, there will be more than one characteristic which the bearer will have. Since an inverse functional object property requires that there be only one input in the form of a characteristic into the bearer, for any characteristic at most one bearer can bear it. As an example, it is possible for multiple people to bear the characteristic of "has a beard." This will mean that the has characteristic relation will have more than a value of 1 or 0. As such, the has characteristic relation should not be inverse functional. 
+```
+
+3. Model the following natural language expressions using terms from BFO and RO; you are welcome to introduce new terms where needed:  
+```
+  (a) Sally has an arm Tuesday but does not have an arm Wednesday.
+  Sally is instance_of object
+Tuesday is instance_of one-dimensional temporal region 
+Wednesday is an instance of one-dimensional temporal region 
+Arm is an instance of fiat object part
+“Sally participates in having at least one arm on Tuesday” is an instance of occurrent 
+“Sally participates in having no arms on Wednesday” is an instance of occurrent
+“Sally participates in having at least one arm on Tuesday precedes Sally participates in having no arms on Wednesday”
+
+  (b) Every liver has some cell as part at all times it exists.
+  Liver has_part_at_all_times Cell
+  
+  (c) John was a child, then an adult, then a senior. 
+  John is an instance_of object
+John's childhood is an instance_of occurrent.
+John's adulthood is an instance_of occurrent.
+John's seniorhood is an instance_of occurrent.
+“John participates in childhood precedes John participates in adulthood which precedes John participates in seniorhood.”
```

Comment:
You'll need to unpack the participation relation and the relationships between the processes and time